howto.md

```diff
@@ -14,11 +14,16 @@ This website is implemented using [github's Pages service](https://pages.github.
 
 ## How to add and edit information
 
-To create or edit files you have to use the [github repository of the HSF website](https://github.com/HEP-SF/hep-sf.github.io/), so you need to be an [HSF github repository](https://github.com/HEP-SF) user. Talk to any member of the startup team.
+For adding information to this page or improving it, we follow the *[pull request](https://help.github.com/articles/using-pull-requests/)* workflow in GitHub.
 
-If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
+Just fork our [HSF website repository](https://github.com/HEP-SF/hep-sf.github.io), edit the
+files you want to edit and open a pull request.
 
-If you only do simple operations like the ones mentioned below, the GitHub interface itself gives you all options to add and edit such files in the browser.
+If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview your submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
+
+If you only do simple operations like the ones mentioned below, the GitHub
+interface itself gives you all options to add and edit such files in the browser.
+So no local checkout is required.
 
```

Comment:
I'd say: `Despite this is not the recommended option, if you are not confortable with Git and you only want to do simple changes, the GitHub web interface allows you to add and edit the files in your browser. In this case, you don't need to have a local clone of your personal fork`on GitHub.`